docs/pages/bare/installing-updates.mdx

```diff
@@ -2,61 +2,61 @@
 title: Installing expo-updates
 ---
 
-import InstallSection from '~/components/plugins/InstallSection';
 import { DiffBlock } from '~/ui/components/Snippet';
 import { Collapsible } from '~/ui/components/Collapsible';
+import { Terminal } from '~/ui/components/Snippet/Blocks/Terminal';
 
-`expo-updates` fetches and manages updates to your app stored on a remote server.
+The expo-updates library fetches and manages updates received from a remote server. It supports [EAS Update](/eas-upadate/introduction), a hosted service that serves updates for projects using the expo-updates library.
```

Comment:
```suggestion
The `expo-updates` library fetches and manages updates received from a remote server. It supports [EAS Update](/eas-upadate/introduction), a hosted service that serves updates for projects using the expo-updates library.
```

We use inline syntax highlight for any library name or package name. also, the rest of the doc does use it for `expo-updates`. I think we should stick to it here too.